packages/did-ethr-resolver/src/implementations/web3Signer/ewJsonRcpSigner.ts

```diff
@@ -0,0 +1,19 @@
+import { providers } from "ethers";
+import { EwSigner } from "./ewSigner";
+
+
+// Example:
+// import detectMetamask from "@metamask/detect-provider";
+// const web3Provider = await detectMetamask()
+// const web3IdentityOwner = Web3IdentityOwner.fromWeb3Provider(web3Provider)
+
+
+// @param web3Provider: an EIP1193 provider
```

Comment:
Should use js-doc comment syntax here with 
```suggestion
/**
 * Example:
 * import detectMetamask from "@metamask/detect-provider";
 * const web3Provider = await detectMetamask()
 * const web3IdentityOwner = Web3IdentityOwner.fromWeb3Provider(web3Provider)
 * @param web3Provider: an EIP1193 provider
 */
```